examples/tutorials/self-paced-training/part-1_federated_learning_introduction/chapter-2_develop_federated_learning_applications/02.1_federated_statistics/federated_statistics_introduction.ipynb

```diff
@@ -6,10 +6,25 @@
    "source": [
```

Comment:
not this file, but some links in 02.0 introduction are not working, we might want to fix those

Reply:
I fixed one, if there are more I can fix those too